### ISSUE TITLE
apt Repository Unusable due to Expired Key Signature

### ISSUE THREAD

**Issue by so-rose** [dimkanovikov/KITScenarist#977]
Dear author,

When installing on Debian 11 via the official `apt` repository (https://software.opensuse.org/download.html?project=home%3AAndnoVember%3AKITScenarist&package=kitscenarist), reports the following error due to an expired signature:

```
Err:2 http://download.opensuse.org/repositories/home:/AndnoVember:/KITScenarist/Debian_11  InRelease      
  The following signatures were invalid: EXPKEYSIG 4B324A81BFC23B5A home:AndnoVember OBS Project <home:AndnoVember@build.opensuse.org>
```

When viewing the key on `keyserver.ubuntu.com` (https://keyserver.ubuntu.com/pks/lookup?search=4B324A81BFC23B5A&fingerprint=on&op=index), I see the following:

```
pub [rsa2048/03ae044f01c4daa33c8356084b324a81bfc23b5a](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x03ae044f01c4daa33c8356084b324a81bfc23b5a) 2016-05-11T17:45:43Z
	 Hash=[92949600adc2954f0d5f5728cb021fb0](https://keyserver.ubuntu.com/pks/lookup?op=hget&search=92949600adc2954f0d5f5728cb021fb0)

uid home:AndnoVember OBS Project <home:AndnoVember@build.opensuse.org>
sig  sig  [4b324a81bfc23b5a](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x4b324a81bfc23b5a) 2016-05-11T17:45:43Z 2018-07-20T17:45:43Z ____________________ [[selfsig]](https://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0x4b324a81bfc23b5a)
sig  sig  [3b3011b76b9d6523](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3b3011b76b9d6523) 2016-05-11T17:45:43Z ____________________ ____________________ [3b3011b76b9d6523](https://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0x3b3011b76b9d6523)
```

**In other words, the signature on the `apt` key expired in July, 2018**. Simply renewing and re-uploading to the key servers of the world would fix the issue.

Thank you for an absolutely wonderful piece of software; I hope you'll consider fixing this bug!

Kind Regards,
so-rose

**Comment by dimkanovikov** [comment]
@aaly11 can you check it?

**Comment by aaly11** [comment]
Привет.
Срок действия ключей домашних проектов на OBS продлевается автоматически, есть такая особенность.
Что нужно сделать?  
1. Удалить ключ с истекшим сроком годности
```
$ sudo apt-key del 4B324A81BFC23B5A
Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
OK
```  
2. Добавить тотже ключ с продлённым сроком годности
```
$ curl -fsSL https://download.opensuse.org/repositories/home:AndnoVember:KITScenarist/Debian_11/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/home_AndnoVember_KITScenarist.gpg > /dev/null
```  
3. Для проверки
```
$ sudo apt-key list | grep -A5 KITScenarist
Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
/etc/apt/trusted.gpg.d/home_AndnoVember_KITScenarist.gpg
--------------------------------------------------------
pub   rsa2048 2016-05-11 [SC] [   годен до: 2025-01-03]
      03AE 044F 01C4 DAA3 3C83  5608 4B32 4A81 BFC2 3B5A
uid         [ неизвестно ] home:AndnoVember OBS Project <home:AndnoVember@build.opensuse.org>

```

**Comment by so-rose** [comment]
That works perfectly! Thanks!
